ProjectLighthouse.Servers.Website/Pages/Moderation/NewCasePage.cshtml.cs

```diff
@@ -37,12 +41,21 @@ public async Task<IActionResult> OnPost(CaseType? type, string? reason, string?
         if (type == null) return this.BadRequest();
         if (affectedId == null) return this.BadRequest();
 
+        UserEntity? affectedUserEntity = await this.Database.Users
+            .FirstOrDefaultAsync(u => u.UserId == affectedId.Value);
+
+        if (affectedUserEntity != null && affectedUserEntity.IsModerator)
```

Comment:
Is it possible to use conditional access here?
`affectedUserEntity?.IsModerator`
Also, this should probably check that `type.AffectsUser()` is true